README.md

```diff
@@ -131,3 +132,63 @@ _**NOTE:** Published versions can be installed from the [Chrome Web Store](https
+            2. The preferences should be tested individually and in combinations to ensure that interactions between the
+               preferences are working properly. For example (Text-To-Speech and Syllabification).
+            3. Multiple web pages should be tested. However, not all preferences will work with all sites. For example,
+               YouTube captions will only work with YouTube videos embedded using the iframe API and providing captions
+               (not autogenerated captions).
+         4. Load Morphic and ensure that logging in users (e.g. [uioPlusCommon](
+            https://github.com/GPII/universal/blob/master/testData/preferences/uioPlusCommon.json5)) applies the
+            preferences to UIO+.
+   3. Increase the "version" number in the [manifest](
+      https://github.com/GPII/gpii-chrome-extension/blob/master/extension/manifest.json#L5) file, and push changes to
+      master.
+2. Create the release package
+   1. Create a release build: `grunt build`
+      1. If making a beta, open the manifest file, located in the newly created build directory, and add a
+      [version name](https://developer.chrome.com/apps/manifest/version#version_name) with the beta release number (e.g.
+      “version_name”:  “0.1.0 beta 10” ). The version name will be displayed instead of the version number in the Chrome
+      Web Store.
+   2. Create a zip archive of the build directory
+3. Publish to the [Chrome Web Store](https://chrome.google.com/webstore/category/extensions)
+   1. Go to the [Developer Dashboard](https://chrome.google.com/webstore/developer/dashboard/g02818309428530539805) on
+      the Chrome Web Store and login using the fluid team account
+   2. On the Developer Dashboard, click “Edit”; located on the right hand side of the UI Options Plus (UIO+) listing
+   3. On the UI Options Plus (UIO+) edit page, click “upload updated package” and upload the zip created in step 2.2
+      above.
+   4. Update the “Detailed description” field as necessary. Similar information is contained in this README.
+   5. Update the screenshots if necessary. They will need to be the exact size requested.
+   6. Until a full release is published we will want to ensure that the “Visibility Options” are set to “Unlisted”. This
+      means that UIO+ will be available for install from the Chrome Web Store, but won't be searchable. It will only be
+      accessible by direct link: [UIO+](
+      https://chrome.google.com/webstore/detail/ui-options-plus-uio%20/okenndailhmikjjfcnmolpaefecbpaek).
+   7. Click "Preview Changes"
+      1. Verify that everything appears correct. Pay particular attention to anything that was changed, e.g. version
+         number/name, descriptions, screenshots and etc.
+   8. When everything appears correct, publish the changes.
+      1. The actual publishing to the Chrome Web Store will take about an hour.
+   9. Tag the master branch with the release (e.g. v0.1.0-beta.10)
```

Comment:
Does it make sense to add one more step after this for creating a release in the github repo based on the new tag?